.jenkinsfile

```diff
@@ -12,6 +12,9 @@ pipeline {
         stage('Setup toolchain') {
             steps {
                 sh 'echo -n "LOCAL CC65 VERSION: " && cc65 --version'
+                script {
+                  currentBuild.description = env.GIT_COMMIT
+                }
```

Comment:
@lydon42 - How do you feel about this change? I feel like it would be convenient to have git hashes more visible alongside the build numbers.

Reply:
I got confirmation from lydon later that this was ok, so proceeding with it.